src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveHandler.cs

```diff
@@ -70,6 +71,10 @@ public async Task<LSP.CompletionItem> HandleRequestAsync(LSP.CompletionItem comp
                 vsCompletionItem.Description = new ClassifiedTextElement(description.TaggedParts
                     .Select(tp => new ClassifiedTextRun(tp.Tag.ToClassificationTypeName(), tp.Text)));
             }
+            else
```

Comment:
Do you not register the VS extension converters in non-VS scenarios? Only asking because it feels a little odd to be checking the type vs. checking a capability

Reply:
We do register them, it just seemed more logical to me that we act and give a response on the data we are given rather than checking the capability